### PR TITLE
LPS-186898 [EVP]  Create Liferay Manager Approval Flow

### PR DESCRIPTION
**Pull request by Mylena-Rodrigues**
Task: [LPS-186898](https://issues.liferay.com/browse/LPS-186898)

### CODE REVIEW

**Comment by Mylena-Rodrigues** [comment]
ci:test:sf

**Comment by eliasantosf** [comment]
Just starting reviewing :)

**Comment by Mylena-Rodrigues** [comment]
ci:test:sf

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-request-filter/index.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+		}
+	);
+
+	const data = await response.json();
+
+	if (!data) {
+		return;
+	}
+
+	return data?.items.map(({id}) => id);
```
[the remark this answers is not in the document]
Comment (Mylena-Rodrigues) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-request-filter/index.js]: Resolved :)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-request-filter/index.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+};
+
+const filterRequests = async () => {
+	const evpUserRequests = await getUserRequests();
+
+	evpRequestRows.forEach((cur_evpRequestRow) => {
+		const cur_evpRequestRow_Id = parseInt(
+			cur_evpRequestRow.querySelector('.component-text').innerText
+		);
+		if (!evpUserRequests.includes(cur_evpRequestRow_Id)) {
```
[the remark this answers is not in the document]
Comment (Mylena-Rodrigues) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-request-filter/index.js]: Resolved :)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-request-kebab-button/index.js [hunk 3]
```diff
@@ -54,84 +91,161 @@ const layerForDendingUpdateStatus = async (message, attribute, key, value) => {
 const getMessage = () => document.querySelector('#messageDescribed').value;
 const getAttributeHidden = () => document.querySelector('#messageDanger');
 
-const openModal = () => {
+const openModal = (btn_option) => {
```
[the remark this answers is not in the document]
Comment (Mylena-Rodrigues) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-request-kebab-button/index.js]: Resolved :)

**Comment by Mylena-Rodrigues** [comment]
ci:test:sf

**Review by silasbsi — APPROVED** [review]
Good job @Mylena-Rodrigues :heavy_check_mark:

**Comment by silasbsi** [comment]
ci:forward

**Comment by Mylena-Rodrigues** [comment]
Link: [brianchandotcom-136905](https://github.com/brianchandotcom/liferay-portal/pull/136905)

**Pull request closed without merging**